update-bot/requirements.txt

```diff
@@ -1,2 +1,3 @@
 httplib2
 mwparserfromhell
+pywikibot>=0.0.dev0
```

Comment:
Shouldn't this rather be `pywikibot>=2.0.dev0`? It makes pip install the same version of pywikibot package but it might be better to have it clearly stated that we are using version 2.0 not just whatever version